SwiftAppTemplate/SwiftAppTemplate/AppDelegate.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  AppDelegate.swift
+//  SwiftAppTemplate
+//
+//  Created by Catriona Scott on 4/25/19.
+//  Copyright © 2019 Catriona Scott. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+
+        // TODO: Create APIClient and call Terminal.setTokenProvider here.
```

Comment:
I think we can remove this comment – better to provide less guidance and make this more generic, I think.

```suggestion
```